pydocstringformatter/configuration/arguments_manager.py

```diff
@@ -0,0 +1,68 @@
+import argparse
+from typing import List
+
+from pydocstringformatter.configuration import (
+    command_line_parsing,
+    formatter_options,
+    toml_parsing,
+)
+from pydocstringformatter.formatting.base import Formatter
+
+
+class ArgumentsManager:
+    """Handler for arugments adding and parsing."""
+
+    def __init__(self, version: str, formatters: List[Formatter]) -> None:
+        # Initialize instance attributes for argument parsing
+        self.parser = argparse.ArgumentParser(prog="pydocstringformatter")
+        self.namespace = argparse.Namespace()
+
+        self.formatters = formatters
+        """List of loaded formatters."""
```

Comment:
Hmm this would load `FORMATTERS` twice as it is also the value passed as `formatters`.

I think we should add this when we allow plug-in formatters.